app/assets/javascripts/angular/controllers/dashboard_ctrl.js

```diff
@@ -82,8 +82,18 @@ angular.module("Prometheus.controllers")
     $scope.nextCycleRedraw();
   };
 
+  $window.history.replaceState({initial: true});
```

Comment:
This records the initial spot in history that the user loads the page. Without using this check, the user could hit "z" to go back in their history to the page BEFORE the dashboard, which is undesired.